### PR TITLE
Update 6LoWPAN mesh namings

### PR DESCRIPTION
**Pull request by artokin**
-Change 6LoWPAN mesh to mesh
-Remove references to Wi-SUN
-Remove reference to additional examples
-Update broken links

### CODE REVIEW

File: docs/reference/technology/mesh/quick_start_intro.md [hunk 1]
```diff
@@ -57,13 +57,3 @@ In a mesh/tree network, all nodes are configured as 6LoWPAN routers.
 In a 6LoWPAN network, RPL protocol is used for forming the routing topology. Every node selects a primary parent for routing so the result looks like a tree.
```
Comment (AnotherButler) [inline docs/reference/technology/mesh/quick_start_intro.md:57]: Query: Should line 39 also change?

Comment (artokin) [reply docs/reference/technology/mesh/quick_start_intro.md:57]: yes it should

Comment (artokin) [reply docs/reference/technology/mesh/quick_start_intro.md:57]: I updated the line #39 and pushed a rebased version in. Would you please recheck?

**Review by AnotherButler — APPROVED** [review]
Looks great, thanks 👍

**Pull request merged**